recipes/osgearth/all/conanfile.py

```diff
@@ -124,7 +124,7 @@ def requirements(self):
         if self.options.with_sqlite3:
             self.requires("sqlite3/3.36.0")
         if self.options.with_draco:
-            self.requires("draco/[>1.3.6]")
+            self.requires("draco")
```

Comment:
Conan allows for overriding dependencies, so we can be explicit here

```suggestion
            self.requires("draco/1.4.3")
```